### DISCUSSION TITLE
ERROR [internal] load metadata for docker.io/timescale/timescaledb:2.4.1-pg13

### DISCUSSION THREAD

**Discussion by cesarebteixeira** [ludeknovy/jtl-reporter#212] [General]
Hello everyone!

I need some help!
i was installing jtlreport and got the above error  and  see image.
Does anyone know how to get around this?
![imageError](https://user-images.githubusercontent.com/3952631/235237351-a3d5d66b-1d76-498c-b793-ece7fc174d97.jpg)

thanks

**Comment by ludeknovy** [comment]
Hello, that’s an docker issue, try to google the error message. But from the description it looks like it was not possible to fetch the socket image due to timeout?
